texture-font.c

```diff
@@ -511,9 +542,16 @@ texture_font_delete( texture_font_t *self )
 {
     size_t i;
     texture_glyph_t *glyph;
+    FT_Error error=0;
 
     assert( self );
 
+    error = FT_Done_Size( self->ft_size );
+    if(error) {
+	freetype_error( error, "FT_Error (%s:%d, code 0x%02x) : %s\n",
+			__FILENAME__, __LINE__, FT_Errors[error].code, FT_Errors[error].message);
+    }
```

Comment:
Crash here in FT_Done_Size if texture_font_close is called before this point, which can happen if a load fails (happening to me right now as the emoji support isn't quite working yet).

Perhaps wrap it in a check:


```suggestion
    if ( self->ft_size ) {
        error = FT_Done_Size( self->ft_size );
        if(error) {
            freetype_error( error );
        }
    }
```

and then in texture_font_close, after the self->face = NULL:

```
self->ft_size = NULL;
```

As it is automatically disposed of by FT_Done_Face?

Not quite sure of the whole object graph here, so I haven't added it to my PR.